flask_security/templates/security/base.html

```diff
@@ -0,0 +1,30 @@
+{% block doc -%}
```

Comment:
can we call it `document` or `base`?

Reply:
I think it's better to keep it the same as in Flask-Bootstrap to avoid potential integration errors.